.github/workflows/_linux-build.yml

```diff
@@ -84,17 +84,29 @@ jobs:
       - name: Checkout PyTorch
         uses: pytorch/pytorch/.github/actions/checkout-pytorch@main
 
+      - name: Get Build Branch
+        uses: ./.github/actions/get-build-branch
+
       - name: Setup Linux
         uses: ./.github/actions/setup-linux
 
+      - name: Checkout pytorch/test-infra to test-infra dir
```

Comment:
Btw, could we move `test-infra` checkout step before the setup linux step?  The reason is that I think we should use `setup-linux` from test-infra eventually https://github.com/pytorch/test-infra/blob/main/.github/actions/setup-linux/action.yml.  So checking out that repo is better to be done before setup-linux I think